### PR TITLE
Add Less comment guidelines

### PR DESCRIPTION
**Pull request by ascott1**
This adds some guidelines to Less comments as discussed in #10.

## Review

@Scotchester, @jimmynotjim, @anselmbradford

### CODE REVIEW

File: css.md [hunk 1]
```diff
@@ -172,6 +171,37 @@ If you find yourself going further, think about re-organizing your rules _(eithe
...
+[Use JSDocs](http://usejsdoc.org/) style comments for comment blocks and `//` for inline comments.
+
+**Example**
+
+```less
+/**
+ * Default button styles
+ * @class .btn
+ * @elements a, button
+*/
```
Comment (Scotchester) [inline css.md]: Should have a space before the asterisk on this last line.

File: css.md [hunk 2]
```diff
@@ -172,6 +171,37 @@ If you find yourself going further, think about re-organizing your rules _(eithe
 - Colors: (use generic variable names like recent cfgov-refresh update)
 - Breakpoints: standard variable names for commonly used breakpoints
 
+### Comments
+
+### JSDocs
```
Comment (Scotchester) [inline css.md]: Should be fourth-level heading.

File: css.md [hunk 3]
```diff
@@ -172,6 +171,37 @@ If you find yourself going further, think about re-organizing your rules _(eithe
...
+*/
+.btn {
+    display: inline-block;
+
+  &:focus {
+    background-color: @btn-bg-hover;
+    outline: 1px dotted @btn-bg;
+    // outline-offset is not supported everywhere but it adds a nice touch where it is
+    outline-offset: 1px;
+  }
```
Comment (Scotchester) [inline css.md]: Indentation here is inconsistent between `display` property and `&:focus` selector.

File: css.md [hunk 4]
```diff
@@ -172,6 +171,37 @@ If you find yourself going further, think about re-organizing your rules _(eithe
...
+}
+```
+
+
+Prefer
+
+### Purpose
```
Comment (Scotchester) [inline css.md]: Should be fourth-level heading.

File: css.md [hunk 5]
```diff
@@ -172,6 +171,37 @@ If you find yourself going further, think about re-organizing your rules _(eithe
...
+    background-color: @btn-bg-hover;
+    outline: 1px dotted @btn-bg;
+    // outline-offset is not supported everywhere but it adds a nice touch where it is
+    outline-offset: 1px;
+  }
+}
+```
+
+
+Prefer
```
Comment (Scotchester) [inline css.md]: Looks like an errant word here.

File: css.md [hunk 6]
```diff
@@ -172,6 +171,37 @@ If you find yourself going further, think about re-organizing your rules _(eithe
...
+
+### Purpose
+Comments **aren't meant to explain what** the code does. Good **code is supposed to be self-explanatory**. If you're thinking of writing a comment to explain what a piece of code does, chances are you need to change the code itself. The exception to that rule is explaining what a regular expression does. Good comments are supposed to **explain why** code does something that may not seem to have a clear-cut purpose.
```
Comment (Scotchester) [inline css.md]: Please add a line break after each sentence.

File: css.md [hunk 7]
```diff
@@ -172,6 +171,37 @@ If you find yourself going further, think about re-organizing your rules _(eithe
...
+
```
Comment (ascott1) [inline css.md]: This shouldn't be here. Whoops.

Comment (Scotchester) [inline css.md]: I think a lot of that statement is still relevant to commenting on Less. Sure you don't want to retain some of it?

File: css.md (hunk 3 above)
Comment (ascott1) [inline css.md]: This should be addressed in the latest push.

File: css.md (hunk 7 above)
Comment (ascott1) [inline css.md]: That's a good point.

Comment (ascott1) [inline css.md]: Added back, but placed before the example and removed the line about regular expressions.

**Comment by Scotchester** [comment]
Nice work! :+1:

**Comment by ascott1** [comment]
Thanks for the review and feedback @Scotchester!

**Pull request merged**